mlir/include/mlir/Dialect/Rock/IR/RockConvInterface.td

```diff
@@ -22,6 +22,27 @@ def RockConvInterface : OpInterface<"RockConvInterface"> {
   let cppNamespace = "::mlir::rock";
 
   let methods = [
+    InterfaceMethod<
```

Comment:
Nite: Rename those to + Shape, like `getFilter()` -> `getFilterShape()`

Reply:
This isn't getting the shape? It's getting the actual value, which is a `TypedValue<ShapedType`